README.md

```diff
@@ -19,10 +19,9 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
 ## Documentation
 
-- [User Documentation](https://www.odoo.com/documentation/14.0/applications.html) - Official user documentation
-- [Development Documentation](https://www.odoo.com/documentation/14.0/) - Official development documentation
-- [Documentation for Install and Maintain](https://www.odoo.com/documentation/14.0/administration.html) - Documentation for Install and Maintain
+- [Product Documentation](https://www.odoo.com/documentation) - Official Documentation - User Docs, Install and Maintain, Developer, Contributing
```

Comment:
Is there a way to continue segmenting and targeting the latest version?

Reply:
I don't know of a way. 

With centralized documentation - does it make sense to segment it in your list?

Reply:
Perfect, as you indicate, please update your pr with the suggested changes.

Thank you for your collaboration